Cargo.toml

```diff
@@ -19,5 +19,9 @@ keywords = ["park", "notify", "thread", "wake", "condition"]
 categories = ["concurrency"]
 exclude = ["/.*"]
 
+[target.'cfg(loom)'.dependencies.loom]
+version = "0.5"
+optional = true
```

Comment:
does it really make sense to make this optional?

Reply:
Requiring a `loom` rustflag makes it so the user would have to go out of their way to enable loom support. This is what we want, since `loom` is an unstable feature. I'll explicitly document this in `Cargo.toml`.

Reply:
uhh but the user needs to pass `--cfg loom` already to rust, right? I guess how dangerous that is depends on how bad the unstability of loom is. So, I guess such opt-in is ok. just make it consistent (by checking for `#[cfg(all(loom, feature = "loom")]

Reply:
As far as I know, the official position is that any API built to use `loom` instead of `libcore/libstd` should be considered radioactive in terms of stability. In fact, using these primitives outside of `loom::model()` will just cause them to panic at runtime.